xbmc/addons/AddonManager.cpp

```diff
@@ -801,6 +801,8 @@ std::string CAddonMgr::GetTranslatedString(const cp_cfg_element_t *root, const c
         translatedValues.insert(std::make_pair(lang, child.value != NULL ? child.value : ""));
       else if (lang == NULL || strcmp(lang, "en") == 0 || strcmp(lang, "en_GB") == 0)
         translatedValues.insert(std::make_pair("en_GB", child.value != NULL ? child.value : ""));
+      else if (lang == NULL || strcmp(lang, "no") == 0)
```

Comment:
The `lang == NULL` doesn't make any sense here because it is already covered by the previous condition.